src/calculator.css

```diff
@@ -42,11 +53,276 @@ li.nav-links:first-child > .active-link {
 }
```

Comment:
- [ ] I'm sure you could find a font more according to the template, please, find it and use it on the title. 🐶 
- [ ] Reviewing the entire template, I think you can use the same template on the book title, book author, and add a new book form too. 🐱 
- [ ] Please add boldness to this title, this will make your title more according to the given template. ⛱️

Reply:
Done!